browser/src/ClinvarVariantsTrack/ClinvarVariantTrack.tsx

```diff
@@ -211,14 +229,32 @@ const ClinvarVariantTrack = ({ referenceGenome, transcripts, variants }: Props)
               {isExpanded ? 'Collapse to bins' : 'Expand to all variants'}
             </Button>
           </ControlRow>
-          <ControlRow>
+          <FilterRow>
             <Checkbox
               id="clinvar-track-in-gnomad"
               label="Only show ClinVar variants that are in gnomAD"
               checked={showOnlyGnomad}
               onChange={setShowOnlyGnomad}
             />
-          </ControlRow>
+            <label htmlFor="star-filtering">
+              Filter by{' '}
+              <Link to="https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/">
```

Comment:
When I click on this it takes me to http://localhost:8008/gene/https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/?dataset=gnomad_r2_1

There is a special link component for external links, for example: https://github.com/broadinstitute/gnomad-browser/blob/70d1916bb76998c9d815c33547d81a8579891c68/browser/src/GenePage/GeneReferences.tsx#L73